addons/mrp/migrations/10.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,419 @@
+mrp          / mrp.workorder            / operation_id (many2one)       : NEW relation: mrp.routing.workcenter
+mrp          / mrp.workorder            / production_id (many2one)      : NEW relation: mrp.production, required: required
+mrp          / mrp.workorder            / qty_produced (float)          : NEW
+mrp          / mrp.workorder            / qty_producing (float)         : NEW
+mrp          / mrp.workorder            / scrap_ids (one2many)          : NEW relation: stock.scrap
+mrp          / mrp.workorder            / state (selection)             : NEW selection_keys: ['cancel', 'done', 'pending', 'progress', 'ready']
+mrp          / mrp.workorder            / time_ids (one2many)           : NEW relation: mrp.workcenter.productivity
+mrp          / mrp.workorder            / workcenter_id (many2one)      : NEW relation: mrp.workcenter, required: required
+# NOTHING TO DO
+
+mrp          / procurement.order        / property_ids (many2many)      : DEL relation: mrp.property
+# NOTHING TO DO
+
+
+mrp          / stock.move               / consumed_for (many2one)       : DEL relation: stock.move
+# DONE: Pre-migration: Renamed to None and copied this column
+
+mrp          / stock.move               / active_move_lot_ids (one2many): NEW relation: stock.move.lots
+mrp          / stock.move               / bom_line_id (many2one)        : NEW relation: mrp.bom.line
+mrp          / stock.move               / consume_unbuild_id (many2one) : NEW relation: mrp.unbuild
+mrp          / stock.move               / move_lot_ids (one2many)       : NEW relation: stock.move.lots
+mrp          / stock.move               / operation_id (many2one)       : NEW relation: mrp.routing.workcenter
+mrp          / stock.move               / quantity_done_store (float)   : NEW
+mrp          / stock.move               / unbuild_id (many2one)         : NEW relation: mrp.unbuild
+mrp          / stock.move               / unit_factor (float)           : NEW
+mrp          / stock.move               / workorder_id (many2one)       : NEW relation: mrp.workorder
+# NOTHING TO DO
+
+
+mrp          / stock.move.lots          / done_wo (boolean)             : NEW
+mrp          / stock.move.lots          / lot_id (many2one)             : NEW relation: stock.production.lot
+mrp          / stock.move.lots          / lot_produced_id (many2one)    : NEW relation: stock.production.lot
+mrp          / stock.move.lots          / lot_produced_qty (float)      : NEW
+mrp          / stock.move.lots          / move_id (many2one)            : NEW relation: stock.move
+mrp          / stock.move.lots          / production_id (many2one)      : NEW relation: mrp.production
+mrp          / stock.move.lots          / quantity (float)              : NEW
+mrp          / stock.move.lots          / quantity_done (float)         : NEW
+mrp          / stock.move.lots          / workorder_id (many2one)       : NEW relation: mrp.workorder
+# NOTHING TO DO
```

Comment:
I think you need to fill this table with historic data.

Reply:
Which historic data? This table is new